core/src/main/java/hudson/FilePath.java

```diff
@@ -2349,6 +2349,19 @@ public int copyRecursiveTo(final String fileMask, final String excludes, final F
      * @since 1.532
      */
     public int copyRecursiveTo(final DirScanner scanner, final FilePath target, final String description) throws IOException, InterruptedException {
+        return copyRecursiveTo(scanner, target, description, GZIP);
+    }
+
+    /**
+     * Copies files according to a specified scanner to a target node.
+     * @param scanner a way of enumerating some files (must be serializable for possible delivery to remote side)
+     * @param target the destination basedir
+     * @param description a description of the fileset, for logging purposes
+     * @param compression compression to use
+     * @return the number of files copied
+     * @since 2.196
```

Comment:
```suggestion
     * @since 2.196
```
```suggestion
     * @since TODO
```

Just in case we miss the weekly

Reply:
Okay, will fix